plugins/explain.py

```diff
@@ -83,6 +83,16 @@ class Explain(BotPlugin):
                      'newcomer Pull Request\n- merge a difficulty/low Pull '
                      'Request\n- review at least a difficulty/low or higher '
                      'Pull Request'
+        'do not ask to ask': 'This is a lazy way of asking your question '
```

Comment:
Please, rephrase this line to be less "You did something quite wrong" to more "It's Ok, you just need to be careful from now on and ask specific questions". Current language might scare newcomers away 😄.